lib/runtime/src/ops/batch_matmul.cc

```diff
@@ -244,18 +244,24 @@ CostMetrics measure_operator_cost(SimEnvFactory const &sim,
   return make_metrics(forward_time, backward_time, sync_time, env);
 }
 
-template <>
-void register_task<BATCHMATMUL_INIT_TASK_ID>() {
+OpTaskSignature init_signature() {
```

Comment:
```suggestion
template <>
OpTaskSignature init_signature<BATCHMATMUL_INIT_TASK_ID>() {
```
same for fwd and bwd